app/models/miq_cockpit_ws_worker/authenticator.rb

```diff
@@ -0,0 +1,53 @@
+class MiqCockpitWsWorker::Authenticator
```

Comment:
Ok, changed MiqCockpitWsWorker::Authenticator to be a module. I did not delegate anything because the only thing that uses methods on MiqCockpitWsWorker::Authenticator is the DRB server which takes the module directly so it didn't seem necessary.